### DISCUSSION TITLE
详细教教我自定义下载吧

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1423] [提问] [answered]
我不会自定义下载找文件（）

**Comment by Chiloven945** [comment]
![image](https://user-images.githubusercontent.com/116699482/224316044-ab4aa714-d544-49c3-8472-c0ed582d3815.png)

**Comment by iPodtouch6557** [comment]
在哪找文件=。=

**Reply by WForst-Breeze** [reply]
您的意思是……您是想问在哪里找资源还是说想知道文件下载完毕后的位置呢🤔

**Comment by WForst-Breeze** [comment, marked as answer]
详细教学请见 #1058
我不太明白您具体指的是什么🤔所以我就提供两种可能的解决方案（这两天电脑不在手边，很抱歉不能提供图片）
1. 如果您想问自定义文件下载完毕后去哪了，可以在百宝箱中找到“打开文件夹”按钮，即可打开安装路径；
2. 如果您想问在哪里找资源、找链接，我们可能也无法提供一个完整的回复，推荐可以去相关网站和视频寻找资源，譬如 MCBBS、Bilibili 和互联网。然后可以继续依照 #1058 进行下载操作。

如果需要补充，请在下方留言

**Reply by iPodtouch6557** [reply]
谢谢

**Reply by WForst-Breeze** [reply]
客气啦，举手之劳😘
